### PR TITLE
adding playbook links and align on alert severity

### PR DESCRIPTION
**Pull request by artherd42**

### CODE REVIEW

**Comment by richardtief** [comment]
I'm not sure if the Federation issues are critical. Maybe only from the perspective of Prometheus down. So a warning might be a better choice.

File: global/prometheus-openstack/alerts/prometheus.alerts [hunk 1]
```diff
@@ -1,14 +1,15 @@
 groups:
 - name: prometheus.alerts
   rules:
-  - alert: OpenstackPrometheusFederationRegions
+  - alert: OpenstackGlobalPrometheusFederationRegions
```
Comment (richardtief) [inline global/prometheus-openstack/alerts/prometheus.alerts:4]:
```suggestion
  - alert: OpenstackGlobalPrometheusFederationFailed
```

**Review by richardtief — CHANGES_REQUESTED** [review]

**Review by richardtief — APPROVED** [review]
Thanks.

**Comment by viennaa** [comment]
Please make sure to squash the commits.

**Pull request merged**